pipeline-maven-database/src/main/java/org/jenkinsci/plugins/pipeline/maven/db/AbstractPipelineMavenPluginDao.java

```diff
@@ -344,6 +344,7 @@ public FormValidation validateConfiguration(Config config) {
                                 }
                             }
                             switch (metaData.getDatabaseMajorVersion()) {
+                                case 15:
```

Comment:
please also add 16, as it is released and already tested

Reply:
Sure!